### DISCUSSION TITLE
Best way to hide cursor when inactive in wayland?

### DISCUSSION THREAD

**Discussion by se7en-x230** [prasanthrangan/hyprdots#898] [General]
What's the best way to hide the cursor after x sec inactive?

**Comment by kRHYME7** [comment]
Hello I read it somewhere in the Hyprland wiki. Try to Browse there I forgot the keyword but I am sure it is there.

**Comment by se7en-x230** [comment]
hyprctl keyword general:cursor_inactive_timeout 3
 
 thanks
